src/index.js

```diff
@@ -40,25 +35,25 @@ var INTERFACE = {
 
     return promise
   },
-  rows: function rows () {
-    return thenWithCancel(INTERFACE.query.apply(null, arguments),
+  rows (...args) {
```

Comment:
Instead of `rows: function rows () {` the function object property shorthand is much terser and clearer.